Packaging/Debian/libobjc2-2.0/debian/rules

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/make -f
+# See debhelper(7) (uncomment to enable)
+# output every command that modifies files on the build system.
+#export DH_VERBOSE = 1
+
+
+# see FEATURE AREAS in dpkg-buildflags(1)
+#export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+
+# see ENVIRONMENT in dpkg-buildflags(1)
+# package maintainers to append CFLAGS
+#export DEB_CFLAGS_MAINT_APPEND  = -Wall -pedantic
+# package maintainers to append LDFLAGS
+#export DEB_LDFLAGS_MAINT_APPEND = -Wl,--as-needed
+
+
+%:
+	dh $@
+
+override_dh_auto_configure:
+	dh_auto_configure -- \
+	-DGNUSTEP_INSTALL_TYPE=NONE \
+	-DCMAKE_C_COMPILER=clang \
+	-DCMAKE_CXX_COMPILER=clang++ \
+	-DCMAKE_C_FLAGS="-I/usr/NextSpace/include -g" \
```

Comment:
I add "-g" option (along with "-DCMAKE_BUILD_TYPE=Release") to be able to build debuginfo on new RedHat systems. Is it a case for Debian also?

Reply:
The tools create -dbgsym packages, so the -g is useful here, yes.